load-testing/audit-service/sendLoginMessages.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
since there are new scripts, does this doc need an update? https://github.com/uc-cdis/gen3-qa/blob/master/load-testing/audit-service/Audit-service.md

Reply:
Yes we do need to update this doc. Since the existing one doesnt include the SQS part of this